### ISSUE TITLE
Compile presets without ffmpeg GPL dependencies

### ISSUE THREAD

**Issue by bakua** [bytedeco/javacpp-presets#260]
Hi. I need to recompile ffmpeg presets version 0.11 for one old project I have. I also need to exclude all GPL licensed dependencies from ffmpeg. I've removed libs stated at https://github.com/bytedeco/javacpp-presets/blob/0.11/ffmpeg/LICENSE.md#compatible-libraries from cppbuild. I am able to compile ffmpeg this way just fine, but when I try to mvn install I run into this exact issue https://groups.google.com/d/msg/javacv/vJSAH_iq4vY/qGUDdfPoFV8J

@saudet  You've adviced to remove postproc.java. But it seems that just removing that file from ffmpeg/src dir is not enough, right? I have to go through all java files and remove all dependencies on postproc.java, is that correct please?

I appreciate any help, thanks :)

**Comment by saudet** [comment]
Yes, that's right, but it's only one or two places that we need to change.

**Comment by bakua** [comment]
Thanks, I've just solved it. All I had to do is run `mvn clean` because maven had `postroc` already cached, so it was trying to parse it even when it was not in src directory any more :) dummy me

Just in case anyone is doing the same, I've removed `src/../postproc.java` and `src/../presets/postproc.java`. Then I removed refference to `postproc.java` from `presets/avfilter.java` in `@Properties` annotation.
